cloudferrylib/os/network/neutron.py

```diff
@@ -279,7 +279,6 @@ def convert_subnets(snet, cloud):
         }
 
         res_hash = network_res.get_resource_hash(result,
-                                                 'name',
```

Comment:
Having `network_name` is good, but I still think we should also have subnet `name` too, it will narrow the number of possible false alarms.